bin-src/qx

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+const path=require("path");
+const updateNotifier = require("update-notifier");
+const pkg = require(path.join(__dirname, "..", "package.json"));
+updateNotifier({pkg}).notify({defer:false});
```

Comment:
IMHO you do not need the update notification here. It's for running the developmnent verion only, is'nt it?

Reply:
yep